quickassist/lookaside/access_layer/src/qat_direct/vfio/qat_mgr_lib.c

```diff
@@ -1297,6 +1297,13 @@ int qat_mgr_build_data(const struct qatmgr_dev_data dev_list[],
                                                       &ring_to_svc_map);
                 if (0 == ret)
                 {
+                    /* Override the ecEdMont capability reported by the kernel.
```

Comment:
NIT. We can start multiline comments with an empty line like in the kernel.

Apart from this, the change can be merged.

Reply:
done